app/views/devise/sessions/new.html.erb

```diff
@@ -1,26 +1,33 @@
-<h2>Log in</h2>
+</html> 
+<head>
+  <link href='https://fonts.googleapis.com/css?family=Graduate' rel='stylesheet' type='text/css'>
+  <link href='https://fonts.googleapis.com/css?family=Roboto' rel='stylesheet' type='text/css'>
+</head>
 
-<%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
-  <div class="field">
-    <%= f.label :email %><br />
-    <%= f.email_field :email, autofocus: true %>
-  </div>
+<body>
+  <div class="background"></div>
```

Comment:
<%= yield %> is in `application.html.erb` by default. `<%= yeild %>`'s purpose is to pull out the content from your view files (such as index.html.erb, show.html.erb etc) and put in the content of the HTML in there.

Does that kind of answer your question? :)